src/libsyntax/print/pprust.rs

```diff
@@ -2270,6 +2270,11 @@ impl<'a> State<'a> {
                 self.print_expr(e)?;
                 word(&mut self.s, "?")?
             }
+            ast::ExprKind::Catch(ref blk) => {
+                self.head("catch")?;
```

Comment:
Oh geeze-- good catch. I didn't think to change that when I added the `do` prefix.